lib/internal/bootstrap/loaders.js

```diff
@@ -96,6 +96,33 @@ const internalBindingWhitelist = new SafeSet([
   'zlib'
 ]);
 
+{
+  const { setPrepareStackTraceCallback } = getInternalBinding('errors');
```

Comment:
There isn't a lot of value in doing this so early, and can you move this to `bootstrap/node.js` instead so that this file still focuses on setting up the loaders? (also ideally only the `native_module` binding should be loaded in this phase)